rpc/client/local/local.go

```diff
@@ -116,6 +116,10 @@ func (c *Local) CheckTx(ctx context.Context, tx types.Tx) (*coretypes.ResultChec
 	return c.env.CheckTx(c.ctx, tx)
 }
 
+func (c *Local) RemoveTx(ctx context.Context, txKey types.TxKey) error {
+	return c.env.Mempool.RemoveTxByKey(txKey)
```

Comment:
It looks like this is returning the internal error directly to the user. I think it would a bit more user friendly if we could create a public error value that users can check against when calling this endpoint, otherwise it's difficult to understand if they should try again.

Reply:
I think that's fine? None of the errors from this would ever be retriable.